DependencyInjection/EverlutionEmailExtension.php

```diff
@@ -39,16 +38,27 @@ public function load(array $configs, ContainerBuilder $container)
 
     private function defineServices(ContainerBuilder $container, array $processedConfig)
```

Comment:
Whole method does not work as intended, since $container->getDefinitions() is empty